desktop-app/app/components/WebView/index.js

```diff
@@ -397,6 +412,46 @@ class WebView extends Component {
     this.webviewRef.current.send('disableInspectorMessage');
   };
 
+  setNetworkThrottlingProfile = ({
+    type,
+    downloadKps,
+    uploadKps,
+    latencyMs,
+  }) => {
+    // TODO : change this when https://github.com/electron/electron/issues/21250 is solved
+    // if (type === 'Online') {
+    //   this.getWebContents().session.disableNetworkEmulation();
+    // } else if (type === 'Offline') {
+    //   this.getWebContents().session.enableNetworkEmulation({offline: true});
+    // } else if (type === 'Custom') {
+    //   const downloadThroughput = downloadKps != null? downloadKps * 128 : undefined;
+    //   const uploadThroughput = uploadKps != null? uploadKps * 128 : undefined;
+    //   this.getWebContents().session.enableNetworkEmulation({offline: false, latency: latencyMs, downloadThroughput, uploadThroughput });
+    // }
+
+    // WORKAROUND
+    if (type === 'Online') {
+      this.dbg.sendCommand('Network.disable');
+    } else if (type === 'Offline') {
+      this.dbg.sendCommand('Network.enable').then(_ => {
+        this.dbg.sendCommand('Network.emulateNetworkConditions', {offline: true, latency: 0, downloadThroughput: -1, uploadThroughput: -1});
+      });
+    } else {
+      const downloadThroughput = downloadKps != null? downloadKps * 128 : -1;
+      const uploadThroughput = uploadKps != null? uploadKps * 128 : -1;
+      const latency = latencyMs || 0;
+      this.dbg.sendCommand('Network.enable').then(_ => {
+        this.dbg.sendCommand('Network.emulateNetworkConditions', {offline: false, latency, downloadThroughput, uploadThroughput});
+      });
+    }
+  }
+
+  clearNetworkCache = () => {
+    this.getWebContents().session.clearCache().then(() => {
+      this.webviewRef.current.reload();
```

Comment:
@jjavierdguezas Should we reload it by default Or just leave it for the user to reload it?

Reply:
done